src/BUILD

```diff
@@ -355,6 +356,13 @@ filegroup(
     ("_nojdk", True),
 ]]
 
+genrule(
+    name = "platforms_archive",
+    srcs = ["@platforms//:srcs"],
+    outs = ["platforms.zip"],
+    cmd = "zip -qX $@ $$(echo $(SRCS) | sort)",
```

Comment:
Is this deterministic? (IIRC we set the timestamps to some constant on the final zip call, but it would be nice to get change pruning in this step, too)

Reply:
Which part worries you? That files coming from the http_archive will change timestamps but not their content? Let's address that on bazelbuild/platforms releases pipeline then?

Reply:
I think it's deterministic, see https://source.bazel.build/bazel/+/master:src/main/java/com/google/devtools/build/lib/bazel/repository/ZipDecompressor.java;l=165